zerver/signals.py

```diff
@@ -34,11 +36,14 @@ def get_device_browser(user_agent: str) -> Optional[str]:
         return "Safari"
     elif "msie" in user_agent or "trident" in user_agent:
         return "Internet Explorer"
+    elif device_browser != "Other":
+        return device_browser
```

Comment:
I would have expected this to _replace_ all the cases above (except potentially the `Zulip` case), rather than being tacked on to the end. Is that something you tried?

Reply:
Some tests were failing as user-agents were not detected by ua_parser because the user-agent strings in the were in lower case. Instead of changing all the strings causing this, I thought adding this at the end would be better. Shall I modify the strings in tests and remove everything else other than `Zulip` ones in `get_device_browser`  and `get_device_os`?